coresimd/powerpc/altivec.rs

```diff
@@ -666,6 +734,18 @@ mod tests {
     use simd::*;
     use stdsimd_test::simd_test;
 
+    #[simd_test(enable = "altivec")]
+    unsafe fn vec_perm_u16x8() {
```

Comment:
there should be one run-time test for every vector type as well

I think that for consistency with how it is done in the other architectures we should name the run-time tests: `test_{intrinsic_name}_{short_vector_types}`, so this becomes `test_vec_perm_u16x8`.

Reply:
I can macro those tests, but it would basically test that transmute works for those types.

Reply:
That would be great, testing these is tedious, but sometimes optimizations happen that break this, and we also scan for assert_instr in `stdsimd-verify`. It would be great if the names of these functions follows a convention so that the code that scans them automatically becomes easier to write. (I followed one convention for vec_add, but maybe there is a better one).